### PR TITLE
Add events/_id endpoint to API (Issue #46)

### PR DESCRIPTION
**Pull request by sdotson**
Hello all!

This is for issue #46 . If there's a preferred process for contributing let me know. I just gave it a go. Let me know what you think!

### CODE REVIEW

File: routes/handlers/event.js [hunk 1]
```diff
@@ -12,9 +12,9 @@ const OPTS_SCHEMA = Joi.object().keys({
 });
 
 exports.get = function (opts, next) {
-  Joi.validate(opts, OPTS_SCHEMA, function (err, query) {
+  Joi.validate(opts.query, OPTS_SCHEMA, function (err, query) {
     if (err) handleError(next, 'validating', err);
-    const filter = createFilter(query.$filter);
+    const filter = createFilter(query.$filter) || opts.params;
```
Comment (aaghevli) [inline routes/handlers/event.js]: Why do we need the || here?

File: routes/index.js [hunk 2]
```diff
@@ -43,7 +54,7 @@ const createRoute = function (method, path, serverMethod) {
     method: method,
     path: path,
     handler: function (request, reply) {
-      serverMethod(request.query, function (error, result) {
```
Comment (aaghevli) [inline routes/index.js]: Good change.

File: routes/handlers/event.js (hunk 1 above)
Comment (sdotson) [inline routes/handlers/event.js]: And this is why we code review. Yep, doesn't need to be there. It was a relic from when I was trying to figure out how it all worked. Removed in the latest commit.

Comment (aaghevli) [inline routes/handlers/event.js]: Great! Merging now...

**Pull request merged**